solver/src/http_solver.rs

```diff
@@ -130,36 +130,43 @@ impl HttpSolver {
             default_fee: 0.0,
         })
     }
-}
 
-#[async_trait::async_trait]
-impl Solver for HttpSolver {
-    async fn solve(&self, orders: Vec<Liquidity>) -> Result<Option<Settlement>> {
+    async fn send(&self, model: &BatchAuctionModel) -> Result<SettledBatchAuctionModel> {
         let mut url = self.base.clone();
         url.set_path("/solve");
         self.config.add_to_query(&mut url);
-        let body = self.create_body(orders.as_slice()).await?;
+        let query = url.query().map(ToString::to_string).unwrap_or_default();
         let mut request = self.client.post(url);
         if let Some(api_key) = &self.api_key {
             request = request.header("X-API-KEY", api_key);
```

Comment:
nit: maybe we can set this header to [sensitve](https://docs.rs/reqwest/0.11.0/reqwest/header/struct.HeaderValue.html#method.set_sensitive) to that it doesn't get leaked in the logs?

Reply:
done